### PR TITLE
Add BIP98: Fast Merkle Trees

### PR DESCRIPTION
**Pull request by maaku**
This contains the revised draft proposal for BIP98: Fast Merkle Trees, which incorporates all the feedback received so far from the developer mailing list and in-person review. This BIP draft is considered complete by its authors.

For context, see the following mailing list discussions:

https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2017-September/014932.html
https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2017-September/014935.html
https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2017-September/015028.html
https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2017-October/015209.html

### CODE REVIEW

**Comment by maaku** [comment]
From Travis:

> Unknown field License-Code in bip-0098.mediawiki at scripts/buildtable.pl line 167, <$F> line 12.

License-Code in this BIP is compliant with BIP-2.I think maybe buildtable.pl is in error?

**Comment by luke-jr** [comment]
> License-Code in this BIP is compliant with BIP-2.I think maybe buildtable.pl is in error?

You're correct.

This is however missing (above Status):

```mediawiki
  Comments-Summary: No comments yet.
  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-0098
```

Please also add a brief backward compatibility section (saying it isn't, and should only be used in/for new rules).

Finally, there should be a simple Copyright section with the full license terms in plain English.

**Comment by luke-jr** [comment]
Why closed?
